.circleci/config.yml

```diff
@@ -31,23 +31,43 @@ aliases:
 
 
 jobs:
-  build:
+  OpenData_DEV:
     <<: *container_config
     parallelism: 1
     environment:
+      VARS_TYPE: OpenData
       DEPLOY_ENV: DEV
     steps: *steps_config
 
-  build_staging:
+  OpenData_STAGING:
     <<: *container_config
     parallelism: 1
     environment:
+      VARS_TYPE: OpenData
+      DEPLOY_ENV: STAGING
+    steps: *steps_config
+
+  Publications_DEV:
+    <<: *container_config
+    parallelism: 1
+    environment:
+      VARS_TYPE: Publications
+      DEPLOY_ENV: DEV
+    steps: *steps_config
+
+  Publications_STAGING:
+    <<: *container_config
+    parallelism: 1
+    environment:
+      VARS_TYPE: Publications
       DEPLOY_ENV: STAGING
     steps: *steps_config
 
 workflows:
   version: 2
   main:
     jobs:
-      - build
-      - build_staging
+      - OpenData_DEV
```

Comment:
look at having dev's run first then trigger staging
https://circleci.com/docs/2.0/configuration-reference/#requires

look at adding cron to trigger once a month at min
https://circleci.com/docs/2.0/configuration-reference/#cron

Reply:
Having staging depend on dev could save some unnecessary runs in cases of failure, but would double the length of the build...is it worthwhile?

A cron trigger should be simple enough, though.

Reply:
circleci Free
1 concurrent job with access to Linux and Windows builds on Small and Medium sizes
we only have 7% left for the next 3 days.
doing dev first should be helpful but this repo might be better off being migrated to github actions to remove the credit limit.